settings.go

```diff
@@ -74,6 +80,28 @@ func (s *Settings) Scope(db *gorm.DB, request *goyave.Request, dest interface{})
 		}
 	}
 
+	if !s.DisableSearch && s.SearchOperator != nil && request.Has("search") {
```

Comment:
Allow `s.SearchOperator` to be `nil` and use a default operator as said earlier.